src/rules/pylint/settings.rs

```diff
@@ -53,27 +54,52 @@ pub struct Options {
     )]
     /// Constant types to ignore when used as "magic values".
     pub allow_magic_value_types: Option<Vec<ConstantType>>,
+    #[option(default = r"5", value_type = "usize", example = r"max_args = 5")]
+    /// Maximum number of arguments for function / method.
+    pub max_args: Option<usize>,
+    #[option(
+        default = r"^_.*|^ignored_|^unused_",
+        value_type = "String",
+        example = r"ignored-argument-names = skip_.*"
+    )]
+    /// Argument names that match this expression will be ignored.
+    pub ignored_argument_names: Option<String>,
```

Comment:
Definitely appreciate you going through the exercise of adding this, but could we instead use `settings.dummy_variable_rgx`? It's kind of nice to have a centralized definition for "ignorable names", and we already use that across a variety of checks.

Reply:
Sure! I will use this. Thanks for pointing out :point_up:

Reply:
No prob! Sorry that you went through the effort of doing this only to roll it back. But hopefully it'll be a more cohesive experience for users.